lib/gollum/public/gollum/javascript/gollum.js.erb

```diff
@@ -440,10 +440,16 @@ $(document).ready(function() {
             sectionHeading.scrollIntoView();
           }
         }
+        previewContent = $('.tabnav-div#preview-content')[0];
         if (window.MathJax != null) {
-          MathJax.startup.defaultReady();
```

Comment:
Caught a bug with this implementation of the mathjax rendering in the preview page: if you switched to the preview pane a second time math wouldn't render!

I copied this implementation from the [mathjax demo](https://www.mathjax.org/#demo) and the way it refreshes constantly.